### PR TITLE
Add fields for primary_url on authority

### PR DESCRIPTION
**Pull request by CloCkWeRX**
First bit of #442

Probably can infer the majority of the homepages from authority.email.split("@").last; but didn't want to stick that in a migration.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -95,17 +95,21 @@
...
     t.boolean "disabled"
-    t.string  "state",                limit: 20
+    t.string  "state",                        limit: 20
     t.string  "email"
     t.integer "population_2011"
     t.text    "last_scraper_run_log"
     t.string  "morph_name"
     t.boolean "write_to_councillors_enabled",             default: false, null: false
+    t.string  "lga_name15"
+    t.string  "superceded_by"
```
Comment (henare) [inline db/schema.rb]: Looks like you've committed extra stuff that shouldn't be in this PR.

File: db/migrate/20160402093711_add_homepage_url.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddHomepageUrl < ActiveRecord::Migration
+  def change
+    add_column :authorities, :primary_url, :string
```
Comment (henare) [inline db/migrate/20160402093711_add_homepage_url.rb]: How about `home_page` instead? I found `primary_url` a bit confusing.

File: db/schema.rb (hunk 1 above)
Comment (CloCkWeRX) [inline db/schema.rb]: Unfortunately that'll keep appearing due to me not dropping / recreating the db from scratch in a number of pull requests - only realised after the first one or two. Silly db/schema, why u not feature branch well.

Comment (henare) [inline db/schema.rb]: In that case you should `git add --patch` and not commit that stuff.

**Pull request closed without merging**